src/layouts/docs/docs-layout.module.css

```diff
@@ -25,31 +25,261 @@
   height: 100%;
   justify-content: center;
   overflow-y: auto;
-  padding: 24px 24px 100px 24px;
+  padding-bottom: 100px;
+  padding-left: 48px;
+  padding-right: 24px;
+  padding-top: 32px;
   position: relative;
 }
 
-.versionSwitcher {
-  height: 18px;
-  width: 346px;
-}
-
 /* TODO: these styles will be changing very soon, this is just to get us started */
 .main {
+  color: #3b3d45;
+  flex-grow: 1;
+  flex-shrink: 1;
```

Comment:
What are these doing?

Reply:
These + the `width` (although I _think_ `flex-shrink` isn't doing anything, need to double check) make the `<main>` fill the space between the sidebar and the sidecar rather than grow too wide for the space. When there are images in the content, the main is too wide and not fully visible. I will add a comment when I check if `flex-shrink` is doing anything!

Reply:
☝️I think this also solves for an edge case I ran into testing out the product landing - if the content in `main` is very narrow (eg a short heading and like "Click something in the sidebar" as the only paragraph of text), with existing styles, `main` doesn't grow to fill the available space, and the content ends up floating in the middle. I think these styles will fix that case too 👍

Reply:
@zchsh I think I know what you're saying, but do double check: what you're referring to is different than the main & sidecar sticking together and centering like this, right?

![image](https://user-images.githubusercontent.com/43934258/147273647-cab8a6ec-e51e-4713-8d57-da2818012871.png)

Reply:
Never mind actually! I think I found what you were talking about. 😊

![image](https://user-images.githubusercontent.com/43934258/147281600-3a74e1b3-1510-447d-b06a-edfa72ac09a5.png)